Build action is broken

```
...
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
...
 package github.com/cenkalti/backoff/v3: cannot find package "github.com/cenkalti/backoff/v3" in any of:
	/opt/hostedtoolcache/go/1.12.16/x64/src/github.com/cenkalti/backoff/v3 (from $GOROOT)
	/home/runner/work/pg_timetable/go/src/github.com/cenkalti/backoff/v3 (from $GOPATH)
...
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.12
+    - name: Set up Go 1.13
       uses: actions/setup-go@v1
       with:
-        go-version: 1.12
+        go-version: 1.13
       id: go
 
     - name: Check out code into the Go module directory

